packages/zowe-explorer/src/globals.ts

```diff
@@ -65,6 +65,9 @@ export const ICON_STATE_CLOSED = "closed";
 export const FILTER_SEARCH = "isFilterSearch";
 export const VSCODE_APPNAME: string[] = ["Visual Studio Code", "VSCodium"];
 export const ROOTPATH = path.join(__dirname, "..", "..");
+export const SETTINGS_VERSION = "zowe.settings.version";
+export const SETTINGS_TEMP_FOLDER_PATH = "zowe.files.temporaryDownloadsFolder.path";
+
```

Comment:
if we are adding these 2 settings to the global constants, should all setting values be global constants as well or none as globals?  If the decision is all should be global setting then we would have to change all references to them in the code to call the global constant.

Reply:
Hmm if that would be the case on making them into global settings would that be out of scope from this item? I do remember @phaumer suggesting this change for the path and version since it would allow easier maintenance for user settings name changes in the future.

Reply:
wouldn't `easier maintenance for user settings name changes in the future` need to cover all user setting names vs just 2 of them?

Reply:
I do think it might be a good idea for maintainability to add the other settings into the constants, as well. It would make sense to do as part of this PR on standardizing setting names.

Reply:
Sounds good then, I will go ahead and make these changes!